src/components/connection_handler/CMakeLists.txt

```diff
@@ -39,20 +38,20 @@ include_directories (
     ${LOG4CXX_INCLUDE_DIRECTORY}
 )
 
-set (SOURCES
-    ${CH_SRC_DIR}/connection_handler_impl.cc
-    ${CH_SRC_DIR}/connection.cc
-    ${CH_SRC_DIR}/device.cc
-    ${CH_SRC_DIR}/heartbeat_monitor.cc
+set(PATHS
+  ${CMAKE_CURRENT_SOURCE_DIR}/include
```

Comment:
@LuxoftAKutsan As far as I understood that is one of the main objective for this refactoring - simplifying the cmake files by using common functions. So my answer there is no guarantee since there is no explicit reference to each source files, but the correct content should be ensured implicitly (by definition), right?